test/integration/scripting_spec.js

```diff
@@ -651,29 +651,35 @@ describe("Interaction", () => {
             await page.type("#\\33 4R", `${name}`, {
               delay: 10,
             });
-            await page.click("[data-annotation-id='41R']");
-            let color = await page.$eval(
-              ref,
-              el => getComputedStyle(el).backgroundColor
-            );
-            expect(color)
-              .withContext(`In ${browserName}`)
-              .toEqual("rgb(255, 0, 0)");
 
-            await page.click("[data-annotation-id='43R']");
-            color = await page.$eval(ref, el => getComputedStyle(el).color);
-            expect(color)
-              .withContext(`In ${browserName}`)
-              .toEqual("rgb(0, 255, 0)");
+            for (const [id, propName, expected] of [
+              [41, "backgroundColor", "rgb(255, 0, 0)"],
+              [43, "color", "rgb(0, 255, 0)"],
+              [44, "border-top-color", "rgb(0, 0, 255)"],
+            ]) {
+              const current = await page.$eval(
+                ref,
+                (el, p) => getComputedStyle(el)[p],
+                propName
+              );
 
-            await page.click("[data-annotation-id='44R']");
-            color = await page.$eval(
-              ref,
-              el => getComputedStyle(el)["border-top-color"]
-            );
-            expect(color)
-              .withContext(`In ${browserName}`)
-              .toEqual("rgb(0, 0, 255)");
+              await page.click(`[data-annotation-id='${id}R']`);
+              await page.waitForFunction(
+                (r, c, p) =>
```

Comment:
In this case I'm guessing that `c` is the colour, but it's simply too unclear what either `r` or `p` is supposed to represent.

Hence here, and above, can we please *not use* one (and two) letter variable names since it's hurting readability of the code quite a bit?

---

Also, there seems to be one "s" too many in the test name, so you probably want to change https://github.com/mozilla/pdf.js/blob/061637d3f4e354042307f2029e79c48ab0e72337/test/integration/scripting_spec.js#L641
to use:
```
"must change colors"
```

Reply:
> In this case I'm guessing that `c` is the colour, but it's simply too unclear what either `r` or `p` is supposed to represent.
> 

Here `r, c, p` stands for `ref, current, propName` which are the final args of `waitForFunction`.

> Hence here, and above, can we please _not use_ one (and two) letter variable names since it's hurting readability of the code quite a bit?
> 

Yes I know that but I can't use some already used names because of the linter...